packages/db/src/search/field-mapping.ts

```diff
@@ -145,15 +151,50 @@ const collectionMapping : { [key: string]: Function } = {
 };
 
 const executionMapping : { [key: string]: Function } = {
+  asyncOperationId: (value?: string) => ({
```

Comment:
This is a field needing join with another table.  Can you move the field to the end, and make a comment as in granuleMapping? `// The following fields require querying other tables`